testing/deployer/sprawl/acl_rules.go

```diff
@@ -102,7 +102,7 @@ func tokenForWorkload(wrk *topology.Workload, overridePolicy *api.ACLPolicy, ent
 		token.TemplatedPolicies = []*api.ACLTemplatedPolicy{{
 			TemplateName: api.ACLTemplatedPolicyWorkloadIdentityName,
 			TemplateVariables: &api.ACLTemplatedPolicyVariables{
-				Name: wrk.ID.Name,
```

Comment:
bug3: We disregarded the user-specified `WorkloadIdentity` field when creating ACL tokens.